Allow wildcard matching in method names

lehni opened the pull request:
This patch adds support for simple wildcard matching in method names, e.g.:

```json
  "mixins": {
    "SetupRemoteMethods": {
      "disableAllExcept": [
        "find",
        "create",
        "findById",
        "deleteById",
        "*.patchAttributes",
        "*.__get__*",
        "*.__create__*",
        "*.__delete__*",
        "*.__findById__*",
        "*.__destroyById__*"
      ]
    }
  }
```

If no wildcards are in use, performance should not degrade noticeably. It would be great if it's deemed worthy to be merged in. Thank you for the very useful mixin!

c3s4r commented:
Hi. Thanks for sending the PR. It looks great... I haven't had time to test it though,... as soon as I get some free time I will test it and merge it.

c3s4r commented:
Could not be merged because branch was deleted, but manually merged

lehni commented:
Apologies for deleting it. I stopped using loopback, so I removed all cloned core repos that I fixed some bugs in a year ago, and accidentally also this one.
